crates/provider/src/fillers/gas.rs

```diff
@@ -266,7 +266,7 @@ mod tests {
         let tx = TransactionRequest {
             from: Some(anvil.addresses()[0]),
             value: Some(U256::from(100)),
-            to: address!("d8dA6BF26964aF9D7eEd9e03E53415D37aA96045").into(),
+            to: Some(address!("d8dA6BF26964aF9D7eEd9e03E53415D37aA96045").into()),
```

Comment:
is the `.into()` required?

Reply:
yeah but its just tests so i didn't want to clutter the diff